docker/urls.py

```diff
@@ -0,0 +1,7 @@
+import os
+from django.conf.urls import include, url
+
+urlpatterns = [
+    url(r'^', include('django_prometheus.urls')),
```

Comment:
These urls are already included by django-container: https://github.com/uw-it-aca/django-container/blob/master/project/base_urls.py#L5